frontend/messages/de.json

```
@@ -918,5 +917,14 @@
 	"author": "Autor",
 	"timestampCannotBeInTheFuture": "Der Zeitstempel darf nicht in der Zukunft liegen",
 	"youCanSetPasswordHere": "Sie können hier Ihr Passwort festlegen.",
+	"failedToContactProvider": "Verbindung zum Anbieter fehlgeschlagen",
+	"userIsNotSSO": "SSO-Anmeldung ist derzeit für Ihr Konto deaktiviert.",
+	"enableSSOHelpText": "SSO als Anmeldeoption für alle Benutzer aktivieren.",
+	"keepLocalLogin": "Von erzwungenem SSO ausschließen",
+	"keepLocalLoginHelpText": "Wenn das globale 'Erzwungene SSO' aktiviert ist, ermöglicht dies dem Benutzer, es zu umgehen und sich bei Bedarf mit einem lokal verwalteten Passwort anzumelden.",
```

Biome (1.9.4)

[error] The key youCanSetPasswordHere was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)